src/main/java/org/sonatype/repository/conan/internal/security/ConanContentValidator.java

```diff
@@ -42,8 +44,13 @@ public String determineContentType(final boolean strictContentTypeValidation,
                                      @Nullable final String contentName,
                                      @Nullable final String declaredContentType) throws IOException
   {
-    if (contentName != null && contentName.endsWith(".tgz")) {
-      return ContentTypes.APPLICATION_GZIP;
+    if (contentName != null) {
+      if (contentName.endsWith(".tgz")) {
+        return ContentTypes.APPLICATION_GZIP;
+      }
+      else if (contentName.endsWith("/conanfile.py")) {
```

Comment:
Question: Why does limiting this just to the conanfile fix the problem? Shouldn't the other files also be identified as python files?

Reply:
As far as I am aware there is only a conanfile.py as part of the packaging which is why I wanted to limited it to just this file.